tonic/src/status.rs

```diff
@@ -107,6 +107,13 @@ pub enum Code {
     Unauthenticated = 16,
 }
 
+impl axum::response::IntoResponse for Status {
+    fn into_response(self) -> axum::response::Response {
+        let (parts, tonic_body) = self.to_http().into_parts();
+        axum::response::Response::from_parts(parts, axum::body::boxed(tonic_body))
+    }
+}
```

Comment:
This makes axum a public dependency on tonic, which I don't think we should have.

Then a breaking release of axum requires tonic to ship a breaking release.

I suggest instead you make your own helper function in your crate. Though it's not quite as elegant.

Reply:
Mhm yea that makes sense, didn't think about that.